mifosng-provider/src/integrationTest/java/org/mifosplatform/integrationtests/common/ImageHelper.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.mifosplatform.integrationtests.common;
+
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.specification.RequestSpecification;
+import com.jayway.restassured.specification.ResponseSpecification;
+import com.sun.jersey.core.util.Base64;
+import org.apache.http.HttpHeaders;
+import org.mifosplatform.infrastructure.core.domain.Base64EncodedImage;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+
+public class ImageHelper {
+
+    private static final String STAFF_IMAGE_URL = "/mifosng-provider/api/v1/staff/";
+    private static final String IMAGES_URI = "/images";
+
+    public static Integer createImageForStaff(final RequestSpecification requestSpec, final ResponseSpecification responseSpec,
+                                              Integer staffId) {
+        System.out.println("---------------------------------CREATING AN IMAGE FOR STAFF---------------------------------------------");
```

Comment:
Log? Or maybe extract to a method that takes care of the message format (valid for all syso statements below)